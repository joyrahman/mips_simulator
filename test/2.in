ADDI R7 R0 0
ADDI R1 R0 1
SUBI R2 R1 10
BEQZ R2 12
ADDI R3 R0 1
ADDI R4 R0 1
SUB R2 R4 R1
SUBI R2 R2 1
BEQZ R2 3
MUL R3 R3 R4
ADDI R4 R4 1
BEQZ R0 -6
ST R3 R7 0
ADDI R7 R7 4
ADDI R1 R1 1
BEQZ R0 -14
HLT 
