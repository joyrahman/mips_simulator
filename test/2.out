374
374
870
496
10
0
362880
10
0
0
36
